README.md

```diff
@@ -3,6 +3,17 @@
 !!!!!! Whenever possible user Collections.* or Arrays.* util classes.
 
 1. Implement Bubble sort for both arrays and collections( 2 approaches). Performance ?.. Can you improve it ?
+First approach: 
```

Comment:
Please do not change readme.md we use it as a template for the exercises. If you want you can add comments to your code.